torch/csrc/jit/runtime/symbolic_shape_registry.h

```diff
@@ -24,7 +24,9 @@ shape.
 
 The shape graph should return a new, unaliased List[int] (or tuple of lists for
 multiple returns) and should not modify any input lists. This allows the shape
-graphs to be composed and executed.
+graphs to be composed and executed. Additionally, when there are multiple tensors
```

Comment:
https://github.com/pytorch/pytorch/pull/77120 don't need this additional requirement